src/functions/build_dependency_packages.py

```diff
@@ -20,19 +20,37 @@ def build_dependency_packages(mpr_rpc_json_data, resultcount):
 
 		depends_packages = []
 
-		for j in ['depends', 'makedepends', 'checkdepends']:
+		if j in [f'{os_codename}_conflicts', f'{os_codename}_makedepends', f'{os_codename}_checkdepends']:
 			depends_packages += get_srcinfo_value(j, True)
+		else:
+			for j in ['depends', 'makedepends', 'checkdepends']:
+				depends_packages += get_srcinfo_value(j, True)
```

Comment:
That's going to check if `j` is **inside** that list, but won't loop over each item.

Regarding what I'm wanting (if the distro-specific value exists, use it, otherwise use the default), I would use something like this:

```python
for j in ['depends', 'makedepends', 'checkdepends']:
  distro_dependencies = get_srcinfo_value(f"{os_codename}_{j}", True)

  if distro_dependencies != []:
    depends_packages += distro_dependencies
  else:
    depends_packages += get_srcinfo_value(j, True)
```

(Don't copy-paste that; the actual code in the files uses tabs instead of spaces, and that'll funk up everything).